src/sql/workbench/browser/modelComponents/button.component.ts

```diff
@@ -158,6 +201,18 @@ export default class ButtonComponent extends ComponentWithIconBase<azdata.Button
 		this.setPropertyFromUI<string>(this.setValueProperties, newValue);
 	}
 
+	public get buttonType(): azdata.ButtonType {
+		if (this.isFile === true) {
+			return 'File' as azdata.ButtonType;
+		} else {
+			return this.getPropertyOrDefault<azdata.ButtonProperties, azdata.ButtonType>((props) => props.buttonType, 'Normal' as azdata.ButtonType);
```

Comment:
This is going to fail now, getPropertyOrDefault doesn't need two generics anymore. You'll probably see it when you merge main into this.

Reply:
Removed generics.